tests/test_file_handling.py

```diff
@@ -0,0 +1,15 @@
+import os
+from datasimulator.graph import Graph
+from dictionaryutils import dictionary
+from datasimulator.file_handling import write_to_file_or_log_error
+
+
+def test_write_to_file_or_log_error(tmpdir):
+    """ asking for tmpdir in the parameter provides an empty test directory for testing """
+    graph = Graph(dictionary, "DEV", "test")
+    graph.generate_nodes_from_dictionary()
+    graph.construct_graph_edges()
+    submission_order = graph.generate_submission_order()
```

Comment:
maybe the `submission_order` should be hardcoded in this unit test, instead of calling the real `generate_submission_order` function, so that only the `write_to_file_or_log_error` logic is tested here

Reply:
yeah I could see about making some sample data

Reply:
hey nvm, i didn't realize it was a list of Nodes, i thought it was a list of strings. This is fine

Reply:
sorry I didn't see this until after pushing the data. I think(?) it should still be fin. If we want me to go back to the nodes let me know and I can revert the change.